### PR TITLE
use ISO timestamps in process title

### PR DESCRIPTION
**Pull request by vetinari**
for easier parsing and with date instead of just hours/minutes

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

File: zmon_worker_monitor/zmon_worker/tasks/main.py [hunk 1]
```diff
@@ -123,8 +123,9 @@ def propartial(func, *args, **kwargs):
 
 
 def setp(check_id, entity, msg):
-    setproctitle.setproctitle('zmon-worker check {} on {} {} {}'.format(check_id, entity, msg,
-                                                                        datetime.now().strftime('%H:%M:%S.%f')))
+    setproctitle.setproctitle(
+        'zmon-worker check {} on {} {} {}'.format(check_id, entity, msg,
+                                                  datetime.now().strftime('%Y-%m-%dT%H:%M:%S.%f')))
```
Comment (mohabusama) [inline zmon_worker_monitor/zmon_worker/tasks/main.py]: Simpler:
```python
datetime.now().isoformat()
```

Comment (femueller) [reply zmon_worker_monitor/zmon_worker/tasks/main.py]: 👍

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**